src/styles/index.scss

```diff
@@ -1,3 +1,21 @@
-body {
+@import 'blocks/var';
+@import 'blocks/catalog';
+@import 'blocks/header';
+@import 'blocks/stars';
+@import 'blocks/card';
+@import 'blocks/logo';
+@import 'blocks/nav';
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
```

Comment:
why do you need disable linter here?

Reply:
cause i set 'roboto' font-style here

Reply:
You can just remove quotes for this font name.